clients/js/src/index.ts

```diff
@@ -286,6 +364,20 @@ export class ChromaClient {
     return await this.api.reset();
   }
 
+  public async version() {
+    const response = await this.api.version();
+    return response.data;
+  }
+
+  public async heartbeat() {
+    const response = await this.api.heartbeat();
+    return response.data["nanosecond heartbeat"];
+  }
+
+  public async persist() {
+    throw new Error("Not implemented in JS client")
```

Comment:
If a user is connected to a backend with duckdb+parquet, do we want this to work?

Reply:
I don't think right now since that is not a formal path we want to support (standing up in-memory chroma as a backend service). What do you think?